src/main/java/com/commercetools/sync/services/StateService.java

```diff
@@ -9,12 +9,10 @@
 
 public interface StateService {
     /**
-     * Given a {@code key}, this method first checks if a cached map of state keys -&gt; ids is not empty.
-     * If not, it returns a completed future that contains an optional that contains what this key maps to in
-     * the cache. If the cache is empty, the method populates the cache with the mapping of all state keys to ids
-     * in the CTP project, by querying the CTP project for all states.
-     *
-     * <p>After that, the method returns a {@link CompletionStage}&lt;{@link Optional}&lt;{@link String}&gt;&gt;
+     * Given a {@code key}, this method first checks if a cached map of state keys -&gt; ids contains the key.
```

Comment:
```suggestion
     * Given a {@code key}, this method first checks if a cached map of State keys -&gt; ids contains the key.
```
to make it consistent with the others 😄